ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/TokenAuthentication/TokenAuthentication.cpp

```diff
@@ -36,11 +36,14 @@ namespace
 using namespace Esri::ArcGISRuntime;
 
 TokenAuthentication::TokenAuthentication(QQuickItem* parent /* = nullptr */):
-  QQuickItem(parent),
-  m_portal(new Portal(portalURL, this)),
-  m_portalItem(new PortalItem(m_portal, itemID, this)),
-  m_map(new Map(m_portalItem, this))
+  QQuickItem(parent)
 {
+  // Create local portal and portalItem objects using global portalURL and itemID variables
```

Comment:
I don't think i'd use the word "global". How about:

```suggestion
  // Create local portal and portalItem objects using constants defined for portalURL and itemID.
```